randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -1707,10 +1707,10 @@ Extra - asset_id: collision_camera_019
               Cross Bomb
               All of the following:
                   # Bomb & Power Bombs, to desync the tunnel bomb blocks from the ledge bomb blocks
-                  Bomb and Power Bombs
+                  Bomb and Lay Power Bomb
```

Comment:
This can stay as is. Adding the template is redundant because Morph is already accounted for.

Reply:
But is Main Power Bomb accounted for?

Reply:
The template checking for main pbs is redundant since logic expect main pbs regardless, then pb tanks after mains is collected. I'm not sure why the template is setup that way. So, yes you don't have to use the template in all cases. As long as morph is accounted for somewhere it's fine.